jobbergate-api/alembic/versions/20211216_120231--d3388a22e0e9_initial_migration_for_jobbergate_in_.py

```diff
@@ -0,0 +1,109 @@
+        sa.Column("application_description", sa.String(), nullable=True),
+        sa.Column("application_owner_email", sa.String(), nullable=False),
+        sa.Column("application_file", sa.String(), nullable=False),
+        sa.Column("application_config", sa.String(), nullable=False),
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("updated_at", sa.DateTime(), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_applications_application_identifier"),
+        "applications",
+        ["application_identifier"],
+        unique=True,
+    )
+    op.create_index(
+        op.f("ix_applications_application_name"), "applications", ["application_name"], unique=False
+    )
+    op.create_index(
+        op.f("ix_applications_application_owner_email"),
+        "applications",
+        ["application_owner_email"],
+        unique=False,
+    )
+    op.create_index(op.f("ix_applications_identifier"), "applications", ["identifier"], unique=True)
+    op.create_table(
+        "job_scripts",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("job_script_name", sa.String(), nullable=False),
+        sa.Column("job_script_description", sa.String(), nullable=True),
+        sa.Column("job_script_data_as_string", sa.String(), nullable=False),
+        sa.Column("job_script_owner_email", sa.String(), nullable=False),
+        sa.Column("application_id", sa.Integer(), nullable=True),
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("updated_at", sa.DateTime(), nullable=False),
+        sa.ForeignKeyConstraint(["application_id"], ["applications.id"],),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(op.f("ix_job_scripts_job_script_name"), "job_scripts", ["job_script_name"], unique=False)
+    op.create_index(
+        op.f("ix_job_scripts_job_script_owner_email"), "job_scripts", ["job_script_owner_email"], unique=False
+    )
+    op.create_table(
+        "job_submissions",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("job_submission_name", sa.String(), nullable=False),
+        sa.Column("job_submission_description", sa.String(), nullable=True),
+        sa.Column("job_submission_owner_email", sa.String(), nullable=False),
+        sa.Column("job_script_id", sa.Integer(), nullable=False),
+        sa.Column("slurm_job_id", sa.Integer(), nullable=True),
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("updated_at", sa.DateTime(), nullable=False),
+        sa.ForeignKeyConstraint(["job_script_id"], ["job_scripts.id"],),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_job_submissions_job_submission_name"),
+        "job_submissions",
+        ["job_submission_name"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_job_submissions_job_submission_owner_email"),
```

Comment:
is it a typo or is it indeed right?

Reply:
it's correct `ix` is a traditional prefix for indices in postgres.